package/README.md

```diff
@@ -14,17 +14,40 @@ Speed up your application using Cluster Cache
 ### Optional Requirements
 * [Igbinary](https://www.php.net/manual/en/book.igbinary.php) - it isn't mandatory but influences on the performance
 
+## SHMOP installation
+
+### Ubuntu / Debian
+
+On the Linux distributions, you can install the extension from the official repo:
+
+```bash
+sudo apt-get update 
+sudo apt-get install php-shmop
+```
+
+### Docker
+You can use [Docker PHP Extension Installer](https://github.com/mlocati/docker-php-extension-installer) to install SHMOP extension in the image:
```

Comment:
It didn't even know this existed. I was thinking about the official install tool for extensions mentioned in the Readme of the official php docker image: https://hub.docker.com/_/php

Especially if shmop doesn't need any additional dependencies (I believe it doesn't), then I would suggest to use the official installer instead of this 3rd party one.

Reply:
I didn't know that